app/workers/enqueue_subject_queue_worker.rb

```diff
@@ -70,4 +66,10 @@ def default_strategy_sms_ids
   def workflow_strategy
     @workflow_strategy ||= workflow.selection_strategy
   end
+
+  def strategy_sms_ids
+    selected_sms_ids.compact
+    rescue Subjects::CellectClient::ConnectionError
```

Comment:
Agree, common style when using def as a begin is to indent the rescue with the def..end